pxtblocks/blocklyloader.ts

```diff
@@ -837,6 +860,68 @@ namespace pxt.blocks {
             e.parentNode.removeChild(e);
     }
 
+    function initVariableArgsBlock(b: B.Block, handlerArgs: pxt.blocks.HandlerArg[]) {
+        U.assert(!(b as MutatingBlock).domToMutation);
+        U.assert(!(b as MutatingBlock).mutationToDom);
+
+        let currentlyVisible = 0;
+        let actuallyVisible = 0;
+
+        let i = b.appendDummyInput();
+
+        let updateShape = () => {
+            if (currentlyVisible === actuallyVisible) {
+                return;
+            }
+
+            if (currentlyVisible > actuallyVisible) {
+                const diff = currentlyVisible - actuallyVisible;
+                for (let j = 0; j < diff; j++) {
+                    const arg = handlerArgs[actuallyVisible + j];
+                    i.insertFieldAt(i.fieldRow.length - 1, new Blockly.FieldVariable(arg.name), "HANDLER_" + arg.name);
+                }
+            }
+            else {
+                let diff = actuallyVisible - currentlyVisible;
+                for (let j = 0; j < diff; j++) {
+                    const arg = handlerArgs[actuallyVisible - j - 1];
+                    i.removeField("HANDLER_" + arg.name);
+                }
+            }
+
+            actuallyVisible = currentlyVisible;
+        };
+
+        i.appendField(new Blockly.FieldImage("/cdn/blockly/media/add.svg", 24, 24, "*", () => {
```

Comment:
Is this patched correctly in the cloud?

Reply:
I think I should be using `pxt.webConfig.commitCdnUrl`. I'll fix it